plugin/src/Ex_HTTP.cc

```diff
@@ -1158,6 +1158,30 @@ BufferWriter& Ex_proxy_rsp_status_reason::format(BufferWriter &w, Spec const &sp
   return bwformat(w, spec, ctx._txn.prsp_hdr().reason());
 }
 /* ------------------------------------------------------------------------------------ */
+class Ex_server_ssn_txn_count : public Extractor {
+public:
+  static constexpr TextView NAME { "server-ssn-txn-count" };
+
+  Rv<ActiveType> validate(Config & cfg, Spec & spec, TextView const& arg) override;
+
+  /// Extract the feature from the @a ctx.
+  Feature extract(Context& ctx, Extractor::Spec const&) override;
+
+  BufferWriter& format(BufferWriter& w, Spec const& spec, Context& ctx) override;
+};
+
+Rv<ActiveType> Ex_server_ssn_txn_count::validate(Config &, Spec &, TextView const&) {
+  return { INTEGER };
+}
+
+Feature Ex_server_ssn_txn_count::extract(Context &ctx, Extractor::Spec const&) {
+  return static_cast<feature_type_for<INTEGER>>(ctx._txn.server_ssn_txn_count());
+}
+
+BufferWriter& Ex_server_ssn_txn_count::format(BufferWriter &w, Spec const &spec, Context &ctx) {
```

Comment:
I've changed the base implementation of `Extractor` so the base implementation of `format` has this code, calling the `extract` method to get the value. Unless the extractor needs special handling, it's best to not implement `format`.